coremltools/converters/mil/mil/ops/defs/iOS15/control_flow.py

```diff
@@ -235,10 +235,8 @@ class select(Operation):
     """
     Return the elements selected from either ``a`` or ``b`` depending on the ``cond``.
 
-    The shape of ``cond``, ``a``, and ``b`` must be broadcastable.
-    You must provide ``a`` and ``b`` together, or provide neither.
-    If you provide neither, the operation returns the indices
-    of ``cond`` that are ``True``.
+    You must provide ``a``, ``b`` and ``cond``. The shape of ``cond``, ``a``,
```

Comment:
Good!
Thanks for fixing this ...
Yeah we are not even testing this functionality,
and looks like in the framework, `a, b` are both required for `select`